src/main/kotlin/gropius/model/issue/Label.kt

```diff
@@ -12,7 +12,7 @@ import java.time.OffsetDateTime
 @GraphQLDescription(
     """Label used to mark Issues with.
     A Label consists of a name, a description and a color.
-    Issues are synced to all IMSProjects of Trackables they are part of.
+    Issues may be synced to all IMSProjects of Trackables they are part of.
```

Comment:
Should we explain here WHEN a label will be synced of do we leave that up to the sync service and do not talk about it in the API

Reply:
Ok